cves/kernel/CVE-2021-3573.yml

```diff
@@ -251,8 +261,8 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: this has nothing to do with i18n
```

Comment:
Elaboration on why i18n doesn't apply would help a reader with no knowledge of the vulnerability understand it better.